### PR TITLE
better exception message for invalid cookie duration config

### LINKED ISSUE #258: set cookie duration failed
File "/Users/rock/large/env/lib/python2.7/site-packages/flask_login.py", line 459, in _update_remember_cookie
    self._set_cookie(response)
  File "/Users/rock/large/env/lib/python2.7/site-packages/flask_login.py", line 481, in _set_cookie
    expires = datetime.utcnow() + duration
TypeError: unsupported operand type(s) for +: 'datetime.datetime' and 'int'

##### config.py ########
CSRF_ENABLED = True
WTF_CSRF_ENABLED = False
REMEMBER_COOKIE_DURATION = 1

### PR DESCRIPTION
**Pull request by alanhamlett**
Fixes #258 and #259.

### CODE REVIEW

File: test_login.py [hunk 1]
```diff
@@ -635,6 +635,13 @@ def test_remember_me_uses_custom_cookie_parameters(self):
             self.assertLess(difference, timedelta(seconds=10), fail_msg)
             self.assertGreater(difference, timedelta(seconds=-10), fail_msg)
 
+    def test_remember_me_invalid_duration_timedelta(self):
+        self.app.config['REMEMBER_COOKIE_DURATION'] = 123
+
+        with self.app.test_client() as c:
+            result = c.get('/login-notch-remember')
+            self.assertEqual(result.status_code, 500)
```
Comment (maxcountryman) [inline test_login.py]: Seems like we aren't really asserting much here.

Comment (alanhamlett) [inline test_login.py]: Yea, do you know how to inspect the thrown exception message?

Comment (maxcountryman) [inline test_login.py]: Could you invoke the underlying method directly in a unit test?

Comment (alanhamlett) [inline test_login.py]: Just added the unit test with `assertRaises`.

**Comment by maxcountryman** [comment]
👍

**Pull request merged**